### ISSUE TITLE
Click on usericon... shows color wheel

### ISSUE THREAD

**Issue by theoryshaw** [JohnMcLear/draw#193]
When i click on....
<li id="usericon" data-key="showusers">

it shows color wheel.

**Comment by ocdtrekkie** [comment]
Yeah, I was going to report this too. (Working on another project at the moment.)

I was guessing that might've been intentional because in Etherpad, users are directly associated with colors, but as EtherDraw lets people select multiple colors anyways, that doesn't make sense. I'd kinda expect to see the same connected user list we see in Etherpad.

**Comment by JohnMcLear** [comment]
This is actually normal and desired behavior although it kinda surprised me..

Basically the colorwheel pops up so you can change your color...  

But yeah I thought it behaved similar to Etherpad but I guess I never made it that way -_-

It's something I can do at some point in the future if I find time I guess, I assume it's not super urgent for anyone?

**Comment by ocdtrekkie** [comment]
Definitely no urgency here, I was just playing with Sandstorm porting and discovered there wasn't really a place to pipe in user identities. 

Ideally I think the user drop down should be as similar to Etherpad as possible. But yeah, no critical need. 

With users using multiple colors in Etherdraw, I'm not sure if it makes sense to tie users to a color though, so much as maybe highlighting elements created by a user if you hover over their name, or maybe having their name hover when you mouse over something they created or something like that.
